test/functional/wallet_import_rescan.py

```diff
@@ -125,12 +151,12 @@ def skip_test_if_missing_module(self):
         self.skip_if_no_wallet()
 
     def setup_network(self):
-        extra_args = [["-addresstype=legacy"] for _ in range(self.num_nodes)]
+        self.extra_args = [[]] * self.num_nodes
```

Comment:
is changing this to `self.extra_args` necessary? Does that member get used again? (I think not, since the nodes are not restarted)

Reply:
No, not necessary, but will make the code robust in case a restart is necessary.